### ISSUE TITLE
Literally none of the Ads are being blocked

### ISSUE THREAD

**Issue by pixincreate** [bromite/bromite#2051]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

101.0.4951.53

### Device architecture

arm64

### Android version

12

### Device model

Nokia 6.1 Plus (LineageOS 19.1 Official)

### Changed flags

No-Flags-Changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

Yes

### Is this bug a crash?

No

### Describe the bug

None of the Ads are being blocked on the latest version of Bromite. The same is even with Incognito. I got a page full of Ads on a single page itself. Screenshots attached below for reference. 

### Steps to reproduce the bug

1. Turn on the phone
2. Select Bromite
3. Type a website that supports Ads. As an example, type car on Google.com
4. You'll see a ton of Ads. 

### Expected behavior

Ads are expected to be blocked on Bromite

### Screenshots

![Screenshot_20220505-063836_Bromite](https://user-images.githubusercontent.com/69745008/166851017-d86d55d6-8767-4927-9fa4-206d06fb327d.png)
![Screenshot_20220505-063845_Bromite](https://user-images.githubusercontent.com/69745008/166851021-3e210ca0-471b-492c-bff6-21be6f3d6b77.png)
![Screenshot_20220505-063957_Bromite](https://user-images.githubusercontent.com/69745008/166851022-1d7f5006-c5f4-4f3f-92cf-286256962833.png)

Some more?

**Comment by csagan5** [comment]
Please report logs with `adb logcat | grep AdBlock`

**Comment by pixincreate** [comment]
I think I need Linux to be installed on my PC or atleast WSL.

**Comment by pixincreate** [comment]
How do I get the logs from a windows PC?

I tried the same on Windows, it's unsupported. Did the same on git bash. It's doing nothing.
```
adb logcat | grep AdBlock > sdcard/log.txt
```
This should get save the logs into the los.txt file, but it's just an empty file.

**Comment by pixincreate** [comment]
I flashed the entire ROM and now the Ads are gone.  
I know it's worth this issue, but I wanna know the reason why it happened. 

I couldn't grab the logcat as it didn't respond on with logs upon running the commands on gitbash.

```
user@user: adb logcat | grep AdBlock
| (blinking cursor for hour long..)```

**Comment by uazo** [comment]
> Any other way to grab?

it is possible by using `chrome://crashes/` and pressing "Generate Report"

**Comment by pixincreate** [comment]
[60b0f484-e89d-4225-a314-34231059d763.zip](https://github.com/bromite/bromite/files/8638896/60b0f484-e89d-4225-a314-34231059d763.zip)
Generated from `chrome://crashes/`.

I don't think this contains AdBlock details. If it does, then great!

**Comment by uazo** [comment]
there he is.
```
05-06 14:36:34.679 32453 32453 I chromium: [INFO:adblock_updater_service.cc(83)] AdBlockUpdaterService: starting up. First update attempt will take place in 5 seconds. Next update attempt will take place in 604800 seconds. 
05-06 14:36:39.680 32453 32453 I chromium: [INFO:adblock_updater_service.cc(132)] AdBlockUpdaterService: MostRecentIndexedVersion = 
05-06 14:36:39.682 32453 32453 W chromium: [WARNING:adblock_updater_service.cc(172)] AdBlockUpdaterService: failed to convert version to time.
05-06 14:36:41.492 32453 32453 W chromium: [WARNING:download_filters_task.cc(156)] DownloadFiltersTask: fetching URL 'HTTP://WEBADDRESS.ELIDED' with method GET (no Last-Modified header)
05-06 14:36:43.127 32453 32453 I chromium: [INFO:download_filters_task.cc(185)] DownloadFiltersTask: downloaded 9416893 bytes in 3443ms from 'HTTP://WEBADDRESS.ELIDED' to '/data/user/0/HTTP://WEBADDRESS.ELIDED' with net_error 0 and error 0
05-06 14:36:43.127 32453 32453 W chromium: [WARNING:adblock_updater_service.cc(246)] AdBlockUpdaterService: invalid Last-Modified header, ignoring version check.
05-06 14:36:43.160 32453 32453 I chromium: [INFO:ruleset_service.cc(479)] RulesetService: valid version: 0
```
@pixincreate can I ask for the default locale of your device? do you have something between bromite and the network, like a vpn or a proxy?

**Comment by pixincreate** [comment]
> there he is.

I actually feared whether I'd a backup of it or not as I flashed the entire ROM 14 hours ago.

> like a vpn or a proxy?

Yea, I use a VPN 24/7.

> default locale

I've modified everything. Device locale is set to Switzerland (by setting Device language to English in Switzerland) and the VPN set to Netherlands (most of the time).

**Comment by csagan5** [comment]
> how long would that previously downloaded script survive? Because I believe I haven't updated the scripts for about more 6 months now and I started seeing ads 3 - 4 days back.

It is not a script, it is a `.dat` filters file; how it works is explained here: https://github.com/bromite/bromite/wiki/AdBlocking#update-mechanism

It survives until there is a more recent update available.
